omod/src/test/java/org/openmrs/module/fhir2/providers/PersonFhirResourceProviderWebTest.java

```diff
@@ -10,15 +10,33 @@
 package org.openmrs.module.fhir2.providers;
```

Comment:
I know this isn't in the model I provided, but it would be nice if this included a test or tests for "complex" queries such as `/Person?given=Harold&gender=male,other`.